test/metabase/events/view_log_test.clj

```diff
@@ -89,32 +89,25 @@
                    :model/DashboardCard _dashcard {:dashboard_id (u/id dashboard) :card_id (u/id card)}]
       (let [dashboard (t2/hydrate dashboard [:dashcards :card])]
         (testing "A basic dashboard read event is recorded in EE, as well as events for its cards"
```

Comment:
We need to update this description to explain that the card events are dispatched separately.

Reply:
updated [ca970d6](https://github.com/metabase/metabase/pull/43102/commits/ca970d626ad4d36951f4a3581e5142b1188a86a8)